src/models/editor.interface.ts

```diff
@@ -1,6 +1,7 @@
 import type { EditorArguments } from './editorArguments.interface';
 import type { EditorValidationResult } from './editorValidationResult.interface';
-
+import type { GridOption } from '../models/gridOption.interface';
```

Comment:
after the CI ran I see that the build is failing, 1 of the reason is that you have duplicate import, you can remove line 3 since `GridOption` is imported in line 4 as well so you have duplicate imports.